UM/Math/Color.py

```diff
@@ -70,6 +78,32 @@ def fromARGB(value):
             (value & 0xff000000) >> 24
         )
 
+    @staticmethod
+    def fromARGBLowBits(value):
+        return Color(
+            (value & 0x000f0000) >> 16,
```

Comment:
I think you can just do `(value >> 16) & 0xf`, so you don't need to calculate where that `f` is in the `hex` value.

Reply:
This is congruent with fromARGB, which already existed, which I then also would have to change (it's right above the added functions).

This is just as many operations, and I think you could argue that it reads a bit better (despite the search for the 'right' bite) because it's closer to the intuition of what's going on.

If you still feel I should change it please let me know.